neuvector-monitor-5.3.yaml

```diff
@@ -0,0 +1,38 @@
+package:
+  name: neuvector-monitor-5.3
+  version: 5.3.2
+  epoch: 0
+  description: NeuVector Full Lifecycle Container Security Platform.
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    provides:
+      - neuvector-monitor=${{package.full-version}}
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - gcc
+      - glibc-dev
+      - make
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/neuvector/neuvector
+      tag: v${{package.version}}
+      expected-commit: edbdcba632835d56dcc92ba86323c8a196471289
+
+  - runs: |
+      make -C monitor
+      mkdir -p ${{targets.contextdir}}/usr/local/bin
```

Comment:
I think the binary can be installed as `/usr/bin/neuvector-monitor` - if something expects it to be `/usr/local/bin/monitor` then we can make a subpackage that creates it as a symlink.

Reply:
scanner requires it to be /usr/local/bin/monitor